src/components/shoppingList/shoppingList.tsx

```diff
@@ -0,0 +1,67 @@
+import { ReactElement, useState, type CSSProperties } from 'react'
+import AnimateHeight from 'react-animate-height'
+import { useColorScheme } from '../../hooks/contexts'
+import styles from './shoppingList.module.css'
+
+interface ShoppingListProps {
+  listId: number
+  title: string
+  children?: ReactElement | ReactElement[]
+}
+
+const ShoppingList = ({ listId, title, children }: ShoppingListProps) => {
+  const [expanded, setExpanded] = useState(false)
+
+  const {
+    schemeColorDarkest,
+    borderColor,
+    textColorPrimary,
+    textColorSecondary,
+    hoverColorDark,
+    schemeColorDark,
+    schemeColorLightest,
+  } = useColorScheme()
+
+  const styleVars = {
+    '--scheme-color': schemeColorDarkest,
+    '--border-color': borderColor,
+    '--text-color-primary': textColorPrimary,
+    '--text-color-secondary': textColorSecondary,
+    '--hover-color': hoverColorDark,
+    '--scheme-color-lighter': schemeColorDark,
+    '--scheme-color-lightest': schemeColorLightest,
+  } as CSSProperties
+
+  const toggleDetails = () => {
+    setExpanded(!expanded)
+  }
+
+  return (
+    <div className={styles.root} style={styleVars}>
+      <div
+        role="button"
```

Comment:
The trigger will either eventually be a button or there will be a good reason it isn't, this is bad practice but is really just for the initial iteration.